src/main/resources/application.properties

```diff
@@ -8,7 +8,16 @@ themoviedb.ord.scheme=https
 themoviedb.ord.host=api.themoviedb.org
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
+spring.jpa.properties.hibernate.format_sql=true
 spring.jpa.hibernate.ddl-auto=none
 
+logging.file.path=logs
+logging.level.ROOT=INFO
+logging.level.info.movietrash=DEBUG
+logging.level.org.springframework.cache=TRACE
+logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
+logging.level.org.hibernate.SQL=TRACE
+logging.level.org.springframework.web.client.RestTemplate=TRACE
```

Comment:
have no idea why do you need debug and trace here (it'll pring all app and system related information - your error log will be owerflowed)

Reply:
deleted